MapboxNavigation/RouteVoiceController.swift

```diff
@@ -106,7 +96,7 @@ open class RouteVoiceController: NSObject, AVSpeechSynthesizerDelegate, AVAudioP
             return
         }
         
-        rerouteSoundPlayer.volume = volume
+        rerouteSoundPlayer.volume = spokenInstructionVolume
```

Comment:
AVAudioPlayer’s volume is relative to the hardware volume, using the same would mean that other audio sources would sound louder (or actually the opposite). For example, set hardware volume to 50%, and AVAudioPlayer.volume to AVAudioSession.outputVolume and AVAudioPlayer will use 25% of maximum volume. I'd suggest setting AVAudioPlayer.volume to 1 or even slightly higher.

Reply:
Also, we have a conflicting PR here -> https://github.com/mapbox/mapbox-navigation-ios/pull/673 but I guess we can wait until this issue is ironed out.

Reply:
@frederoni if it's set to 1, while it always ignore the system volume?

Reply:
@bsudekum no, it will strictly follow the system’s current volume.